module-1/cdk/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "cdk",
```

Comment:
This doesn't seem like the default package.json generated from "cdk init". It has lots of stuff that's not related to the workshop